docs/en/latest/plugins/limit-count.md

```diff
@@ -46,7 +46,10 @@ The `limit-count` Plugin limits the number of requests to your service by a give
 | group                   | string  | False                                     |               | non-empty                              | Group to share the counter with. Routes configured with the same group will share the counter.                                                                                                                                                                                                                                                                                                                                                                                                       |
 | redis_host              | string  | required when `policy` is `redis`         |               |                                        | Address of the Redis server. Used when the `policy` attribute is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | redis_port              | integer | False                                     | 6379          | [1,...]                                | Port of the Redis server. Used when the `policy` attribute is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| redis_username          | string  | False                                     |               |                                        | Username of the Redis server. Used when the `policy` is set to `redis`.                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Could you explain in more detail?  It's for authenticating the current connection (redis version >= 6.0), use password only if Redis versions < 6